app/assets/javascripts/comment.js

```diff
@@ -0,0 +1,12 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
+// You can use CoffeeScript in this file: http://coffeescript.org/
+$(document).ready(function() {
```

Comment:
Might need to pass event into function. Might be completely wrong. Ran into an issue with firefox in the past where this ended up being the cause.

http://stackoverflow.com/questions/20522887/referenceerror-event-is-not-defined-error-in-firefox